src/lib/proposer/proposer.ml

```diff
@@ -6,6 +6,12 @@ open O1trace
 module type Inputs_intf = sig
   include Protocols.Coda_pow.Inputs_intf
 
+  module Transition_frontier : sig
```

Comment:
You should probably just use the full signature to follow the pattern, but I will still approve for now.